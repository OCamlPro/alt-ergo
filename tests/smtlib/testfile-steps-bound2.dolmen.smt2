(set-option :steps-bound 2)
(set-logic ALL)

(declare-const x Int)
(assert (= (* x x) 3))
; Should answer 'unknown'
(check-sat)
(get-info :reason-unknown)

(reset)

(set-option :steps-bound 100)
(set-logic ALL)
(declare-const x Int)
(assert (= (* x x) 3))
; Should answer 'unsat'
(check-sat)