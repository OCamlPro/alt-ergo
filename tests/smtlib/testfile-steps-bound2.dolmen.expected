
unknown
(:reason-unknown (:step-limit 3))

unsat
